site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+<table>
+  <thead>
+    <tr>
+      <th></th>
+      <th><strong>Before</strong></th>
+      <th><strong>After</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Data</strong></td>
+      <td>15.4 MB</td>
+      <td>6.1 MB</td>
+    </tr>
+    <tr>
+      <td><strong>Total Blocking Time</strong></td>
+      <td>3.2 seconds</td>
+      <td>1.6 seconds</td>
+    </tr>
+  </tbody>
+</table>
+
+{% YouTube id='Ptf-eK3R3nA' %}
+
+To learn more about this work, see our Explainer and blink-dev [intent-to-experiment thread](https://groups.google.com/a/chromium.org/g/blink-dev/c/PhLkO3KITyw) and [experiment extension](https://groups.google.com/a/chromium.org/g/blink-dev/c/DjYa9xaoHhs).
+
+2.  **Targeted third-party Throttling**
+
+Third-party scripts are often added by various teams without holistic oversight processes. The engineering team at [The Telegraph](https://medium.com/the-telegraph-engineering/improving-third-party-web-performance-at-the-telegraph-a0a1000be5) articulated that "Everyone wants 'that tag' on a page which will make the organization money". This can continuously increase the burden of managing the performance impact.
+
+Targeted 3P throttling proposes to throttle very specific kinds of third-parties to mitigate their impact. This would allow browsers to load key content + critical 3Ps early, while those safe to load later are throttled.
+
+{% YouTube id='oU-TRpghG20' %}
+
+We are also working on enhancing RUM APIs to include information that would be relevant in assessing third-party performance. Enhancements include the following:
+
+3.  iframe reporting: We are working on solutions that can leverage the [Performance Timeline](https://developer.mozilla.org/docs/Web/API/Performance_Timeline) API for cross-frame reporting. This would allow authors of the top-level page to inspect performance data for a cooperating third-party iframe that is embedded on the page.
+
+4.  Long Task attribution: The [Long Tasks API](https://w3c.github.io/longtasks/#dom-performancelongtasktiming-attribution) in RUM will help site owners identify long tasks that tie up the main thread for a long time and delay interaction.
+
+# Further updates
+
+We are still experimenting with many such ideas and hope to publish GitHub explainers and specification drafts for changes as we move along. Third-parties and site-owners who wish to partner with us or leave feedback can contribute to discussions through these. Third-parties can also start focussing on optimizing for Core Web Vitals and INP metrics to ensure that poor CWV/INP data is not attributed to them.
+
+For now, those actively looking for updates can refer to posts on the [blink-dev](https://groups.google.com/a/chromium.org/g/blink-dev) group.
+
+*With special thanks to Leena Sohoni-Kasture, Jeremy Wagner, Gilberto Cocchi, Kenji Baheux, Alex N. Jose, Melissa Mitchell, Yoav Weiss, Kouhei Ueno, Kentaro Hara for their feedback and contributions.*
```

Comment:
```suggestion
*With special thanks to Leena Sohoni-Kasture, Jeremy Wagner, Gilberto Cocchi, Kenji Baheux, Alex N. Jose, Melissa Mitchell, Yoav Weiss, Kouhei Ueno, Kentaro Hara for their feedback and contributions.*

```

Reply:
This newline is just to get @chrome-devrel-review-bot to stop complaining 😅

Reply:
Thanks for the fix! :)